apps/server/src/modules/learnroom/service/common-cartridge-export.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId, Lesson } from '@shared/domain';
+import { LessonService } from '@src/modules/lesson/service';
+import { CourseService } from './course.service';
+import { ICommonCartridgeOrganizationProps, CommonCartridgeFileBuilder } from '../common-cartridge';
+
+@Injectable()
+export class CommonCartridgeExportService {
+	constructor(private readonly courseService: CourseService, private readonly lessonService: LessonService) {}
+
+	async exportCourse(courseId: EntityId): Promise<Buffer> {
+		const course = await this.courseService.findById(courseId);
+		const [lessons] = await this.lessonService.findByCourseIds([courseId]);
+		return new CommonCartridgeFileBuilder({
+			identifier: `i${course.id}`,
+			title: course.name,
+		})
+			.addOrganizationItems(this.mapLessonsToOrganizationItems(lessons))
+			.addResourceItems({
+				identifier: 'placeholder-identifier',
```

Comment:
To have XSD validity, we must prefix the id with a character. We may stay with the i_ solution you had added before.